### ISSUE TITLE
astroquery sometimes works fine with query result but most of the time not

### ISSUE THREAD

**Issue by YouCodeLikeAGirl** [astropy/astroquery#2139]
```
  File "C:\Users\..\AppData\Local\Programs\Python\Python36\lib\site-packages\astroquery\utils\class_or_instance.py", line 25, in f
    return self.fn(obj, *args, **kwds)
  File "C:\Users\..\AppData\Local\Programs\Python\Python36\lib\site-packages\astroquery\utils\process_asyncs.py", line 29, in newmethod
    result = self._parse_result(response, verbose=verbose)
  File "C:\Users\..\AppData\Local\Programs\Python\Python36\lib\site-packages\astroquery\simbad\core.py", line 1066, in _parse_result
    "Exception: " + str(ex))
astroquery.exceptions.TableParseError: Failed to parse SIMBAD result! The raw response can be found in self.last_response, and the error in self.last_table_parse_error. The attempted parsed result is in self.last_parsed_result.
 Exception: 7:115: no element found
```

Sometimes I can receive query result but other times I use exact same query with above error. Could you please investigate the reason and provide a fix?

Thank you in advance.

**Comment by bsipocz** [comment]
Please post a full example you are trying to run, it would help us reproduce the issue and debug.

**Comment by bsipocz** [comment]
I think this has been resolved in #2187. Regarding the need for better documentation for the caching, it already has issues opened, so for that purpose, there is no need to keep this one open, too.

**Comment by emirkmo** [comment]
> rm ~/.astropy/cache/astroquery/Simbad/*.pickle

Ran into a similar issue that relied on an older version of Astropy (v4.3) and this is what solved it, thanks.
